### ISSUE TITLE
Request for License Change or Dual Licensing

### ISSUE THREAD

**Issue by ghost1372** [ikas-mc/ContextMenuForWindows11#186]
Hi @ikas-mc , I am using your library in my own [DevWinUI Library](https://github.com/ghost1372/DevWinUI/tree/main/dev/DevWinUI.ContextMenu).  and due to concerns about the license it seems I cannot use your library.

https://github.com/ghost1372/DevWinUI/issues/11#issuecomment-2525159988

I use the MIT license and you use the LGPL.
Is it possible for you to change the license of this library to MIT? Or any license that allows it to be used in another MIT repository?

If changing the license is not possible, can you alternatively, offer a dual-license option (MIT and LGPL-3.0)?

since i am using only [Service Classes](https://github.com/ikas-mc/ContextMenuForWindows11/tree/main/ContextMenuCustom/ContextMenuCustomApp/Service) and **ContextMenuCustomHost.dll** file, can you only provide MIT License for these?

I respect the work you’ve put into this library and understand the importance of licensing. 
I would be very grateful if you would agree to this request.

**Comment by ikas-mc** [comment]
You can use ContextMenuCustomHost.dll under LGPL

The service code, you can just "copy" them

**Comment by ghost1372** [comment]
Thank you

**Comment by ghost1372** [comment]
@ikas-mc  Does this mean I can't use **ContextMenuCustomHost.dll** in my MIT Licensed Repo?

**Comment by ikas-mc** [comment]
> @ikas-mc  Does this mean I can't use **ContextMenuCustomHost.dll** in my MIT Licensed Repo?

yes, the dll is a completely independent extension and no link required

**Comment by ghost1372** [comment]
> > @ikas-mc  Does this mean I can't use **ContextMenuCustomHost.dll** in my MIT Licensed Repo?
> 
> yes, the dll is a completely independent extension and no link required

So how i can use it? Can you provide as a nuget package?

**Comment by ikas-mc** [comment]
sorry, i can't do it now.

You can create a nuget package to upload by yourself.
